blog/2021-04-20-why-rust-powers-core-sdk.md

```diff
@@ -28,15 +28,15 @@ In this post, we'll dive into the points mentioned above. We will also explain w
 ### What's complex about it?
 From a 10,000 foot view, a Temporal worker follows this algorithm when running a workflow:
 
-1. Long poll the server for workflow tasks (i.e.: Server says "I need you to run the user's workflow code").
+1. Long poll the server for workflow tasks (i.e., Server says "I need you to run the user's workflow code").
```

Comment:
```suggestion
1. Long poll the server for workflow tasks (for example, Server says "I need you to run the user's workflow code").
```
In this case, "e.g." would be correct, not "i.e."—but our style calls for avoiding both. See "i.e." in https://developers.google.com/style/word-list#letter-i.

Reply:
thanks!